CollAction/Controllers/ProjectsController.cs

```diff
@@ -163,7 +163,7 @@ public ViewResult StartInfo()
                 "Hi!<br>" +
                 "<br>" +
                 "Thanks for submitting a project on www.collaction.org!<br>" +
-                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
+                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
```

Comment:
My browser can't really view the character you replaced "-" and "’" with. Unicode error, just shows as a questionmark. Please undo?